repo/packages/K/kafka/0/package.json

```diff
@@ -12,5 +12,17 @@
     "icon-large": "https://d1vubr0evspla.cloudfront.net/img/icon-large.png",
     "screenshots": ["https://d1vubr0evspla.cloudfront.net/img/screen-0-kafka.png"]
   },
-  "postInstallNotes": "Kafka DCOS Service is installed:\n  docs   - https://github.com/mesos/kafka\n  issues - https://github.com/mesos/kafka/issues"
```

Comment:
Could we please have some mention of resource requirements in the preInstall notes?